charts/internallb-webhook-admission-controller/templates/internallb-webhook-admission-controller-deployment.yaml

```diff
@@ -0,0 +1,49 @@
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ template "fullname" . }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "fullname" . }}
+        chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+        release: "{{ .Release.Name }}"
+        releaseRevision: "{{ .Release.Revision }}"
+        heritage: "{{ .Release.Service }}"
+    spec:
+      serviceAccountName: "{{ .Values.controller.serviceAccount }}"
+      containers:
+      - name: webhook
+        image: {{ .Values.controller.image }}:{{ .Values.controller.imageTag }}
+        imagePullPolicy: {{ .Values.controller.imagePullPolicy }}
+        resources:
+          requests:
+            cpu: 100m
+            memory: 20Mi
+          limits:
+            cpu: 100m
+            memory: 30Mi
+        ports:
+        - containerPort: {{ .Values.controller.service.targetPort }}
+        volumeMounts:
+        - name: certs
+          mountPath: /var/run/internallb-webhook-admission-controller
+          readOnly: true
+      volumes:
+      - name: certs
+        secret:
+          secretName: {{ template "fullname" . }}-cert
+          {{- if .Values.controller.tls.requestHeaderCA }}
```

Comment:
question: what does this part with the key / path do? won't this happen automatically if `requestheader-ca.crt` is in the secret?

Reply:
Good catch. This is pointless. Will remove